### PR TITLE
[docs][cloud] Clarify scaling for Region FT

### PR DESCRIPTION
**Pull request by ddhodge**
Clarify scaling operation for clusters with region level fault tolerance

@netlify /preview/yugabyte-cloud/cloud-clusters/configure-clusters/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-clusters/configure-clusters.md [hunk 1]
```diff
@@ -52,6 +52,8 @@ You can scale multi-node single-region clusters horizontally and vertically, as
 
 To add or remove read replicas, refer to [Read replicas](../managed-read-replica/).
 
+For Availability zone-level fault tolerant clusters, you must scale nodes in increments of 3.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-clusters/configure-clusters.md:55]: Shouldn't it be either Availability Zone or availability zone?

File: docs/content/preview/yugabyte-cloud/cloud-clusters/configure-clusters.md [hunk 2]
```diff
@@ -102,7 +108,7 @@ New regions must be deployed in a VPC. New regions have the same fault tolerance
 
 Before you can delete a region, you must drop all the tablespaces located in the region. You can't delete the primary region.
 
-For availability zone-level fault tolerant clusters, you must scale nodes in increments of 3.
+For Availability zone-level fault tolerant clusters, you must scale nodes in each region in increments of 3.
```
Comment (lizayugabyte) [inline docs/content/preview/yugabyte-cloud/cloud-clusters/configure-clusters.md:111]: same comment

**Review by lizayugabyte — APPROVED** [review]

File: docs/content/preview/faq/yugabytedb-managed-faq.md [hunk 3]
```diff
@@ -150,11 +150,12 @@ YugabyteDB Managed uses a shared responsibility model for security. For more inf
...
 
-The Fault Tolerance of a cluster determines how resilient the cluster is to node and availability zone failures and, by extension, the cluster configuration. You can configure clusters with the following fault tolerances in YugabyteDB Managed:
+The Fault Tolerance of a cluster determines how resilient the cluster is to failures and, by extension, the cluster configuration. You can configure clusters with the following fault tolerances in YugabyteDB Managed:
 
-- **Availability Zone Level** - a minimum of 3 nodes spread across multiple availability zones with a RF of 3. YugabyteDB can continue to do reads and writes even in case of an availability zone failure. This configuration provides the maximum protection for a data center failure. Recommended for production deployments. For horizontal scaling, nodes are scaled in increments of 3.
+- **Region Level** - a minimum of 3 nodes spread across 3 regions with a RF of 3. YugabyteDB can continue to do reads and writes even in case of an region failure. This configuration provides the maximum protection for a region failure. Recommended for production deployments. For horizontal scaling, nodes are scaled in increments of 3.
```
Comment (mrajcevic01) [inline docs/content/preview/faq/yugabytedb-managed-faq.md:157]: Is there a reason we call out "recommended for production deployments" ? Setting the FT is based on their application needs. Nothing wrong with running node-level FT if they do not require anything greater.

File: docs/content/preview/faq/yugabytedb-managed-faq.md [hunk 4]
```diff
@@ -150,11 +150,12 @@ YugabyteDB Managed uses a shared responsibility model for security. For more inf
...
+- **Availability Zone Level** - a minimum of 3 nodes spread across multiple availability zones with a RF of 3. YugabyteDB can continue to do reads and writes even in case of an availability zone failure. This configuration provides the protection for a data center failure. Recommended for production deployments. For horizontal scaling, nodes are scaled in increments of 3.
 - **Node Level** - a minimum of 3 nodes deployed in a single availability zone with a [replication factor](../../architecture/docdb-replication/replication/) (RF) of 3. YugabyteDB can continue to do reads and writes even in case of a node failure, but this configuration is not resilient to availability zone outages. For horizontal scaling, you can scale nodes in increments of 1.
```
Comment (mrajcevic01) [inline docs/content/preview/faq/yugabytedb-managed-faq.md:159]: If we are going to point to RF docs lets do it higher up in this section, when it is first mentioned with region-level FT

File: docs/content/preview/faq/yugabytedb-managed-faq.md (hunk 3 above)
Comment (mrajcevic01) [reply docs/content/preview/faq/yugabytedb-managed-faq.md:157]: I saw a below edit that has this phrased as "For production clusters, a minimum of Availability Zone Level is recommended. Whether you choose Region or Availability Zone Level depends on your application architecture, design, and latency requirements." in the create-clusters-overview section. This type of wording at the end of the first sentence may be less confusing than included "recommended for production deployments" in the list

**Pull request merged**